### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by swarnim151**

### CODE REVIEW

File: src/grid_generator.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  const [grid_list, setgridlist] = useState([]);
+
+  useEffect(load_grid, []);
+
+  async function load_grid() {
+    const firestore_grid = await Firestore.get.gridsCreatedByUser(
+      "kricheso@google.com",
+      "kricheso@google.com"
+    );
+
```
Comment (kricheso) [inline src/grid_generator.js:33]: Handle null gracefully.

File: src/grid_generator.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    setgridlist(firestore_grid);
+  }
+
+  return (
+    <div className="board">
+      {grid_list.map((grid, i) => {
```
Comment (kricheso) [inline src/grid_generator.js:40]: if grid_list is null here, the app will crash

File: src/ds.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import CardMedia from "@material-ui/core/CardMedia";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+
+class Grid extends Component {
```
Comment (kricheso) [inline src/ds.js:12]: Not quite sure what this file is doing... Regardless, I do not see the point of having hard coded values like "kricheso@google.com".

**Review by kricheso — CHANGES_REQUESTED** [review]
In general, good work. Please resolve merge conflicts.

**Review by kricheso — APPROVED** [review]

File: src/grid_generator.js (hunk 2 above)
Comment (Azeezah) [reply src/grid_generator.js:40]: Fixed.

**Pull request merged**